coalib/bearlib/languages/definitions/shell.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class shell:
+    aliases = 'bash', 'sh'
+
+    extensions = '.sh', '.bash', '.zsh'
+    comment_delimiter = '#'
+    multiline_comment_delimiters = {': '': '''}
+    string_delimiters = {'"': '"', "'": "'"}
+    multiline_string_delimiters = string_delimiters
+    encapsulators = {'[': ']', '{': '}'}
+    keywords = [
+        'if', 'then', 'else', 'elif', 'fi', 'case', 'coproc', 'esac',
+        'for', 'select', 'while', 'until', 'do', 'done', 'in',
+        'function', 'time']
```

Comment:
`time` isn't  a shell command, it's a POSIX utility.

Reply:
https://man.openbsd.org/time
https://linux.die.net/man/1/time
https://www.freebsd.org/cgi/man.cgi?query=time